easyconfigs/h/HDFView/HDFView-3.2.0-Java-17.0.6-centos8.eb

```
@@ -0,0 +1,35 @@
+easyblock = 'Binary'
+
+name = 'HDFView'
+version = '3.2.0'
+local_label = 'centos8'
+versionsuffix = '-Java-%%(javaver)s-%s' % local_label
+
+homepage = 'https://support.hdfgroup.org/products/java/hdfview/'
+description = "HDFView is a visual tool for browsing and editing HDF4 and HDF5 files."
+
+toolchain = SYSTEM
+
+source_urls = ['https://support.hdfgroup.org/ftp/HDF5/releases/HDF-JAVA/%(namelower)s-%(version)s/bin']
+sources = ['%%(name)s-%%(version)s-%s_64.tar.gz' % local_label]
+checksums = ['205396d3699e2b7fdfeb37034ef8ee296068122b23283184b5d71b508ecbf1e1']
+
+dependencies = [
+    ('Java', '17.0.6'),
```

```suggestion
    ('Java', '17'),
```

We should use the Java 17 wrapper. This will mean the filename should change as well (`HDFView-3.2.0-Java-17-centos8.eb`).

---

Thanks - done.